### PR TITLE
UI tests for V0.2

### PR DESCRIPTION
**Pull request by mattheuslee**
Pull request for all ongoing UI test work so far, for merging at the end of V0.2.

### CODE REVIEW

**Comment by ChuaPingChan** [comment]
Hmm you can merge into the main branch along the way too. Incremental integration is better than merging at the end (in one big bang) right?

**Comment by mattheuslee** [comment]
That's possible, but that would mean merging when the feature isn't complete yet? Like merging but still leaving the PR open?

**Comment by ChuaPingChan** [comment]
Oh. Then nope lol. I didn't look through cuz too many lines and assumed it's ready.

**Comment by ChuaPingChan** [comment]
Sorry I accidentally pressed `Close and comment`.

**Comment by mattheuslee** [comment]
Fix https://github.com/CS2103JUN2017-T2/main/issues/61

**Comment by mattheuslee** [comment]
@CS2103JUN2017-T2/developers reviews?

**Review by ChuaPingChan — APPROVED** [review]

File: src/test/java/guitests/DeleteCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+     * Runs the delete command to delete the floating task at {@code index} and confirms the result is correct.
+     * @param currentList A copy of the current list of floating tasks (before deletion).
+     */
+    private void assertDeleteFloatingTaskSuccess(Index index, final Entry[] currentList) {
+        Entry entryToDelete = currentList[index.getZeroBased()];
+        Entry[] expectedRemainder = TestUtil.removeEntryFromList(currentList, index);
+
+        commandBox.runCommand(EntryUtil.getFloatingTaskDeleteByIndexCommand(index));
+
+        //confirm the list now contains all previous persons except the deleted person
```
Comment (kevinLamKB) [inline src/test/java/guitests/DeleteCommandTest.java]: im guessing this comment should be entries instead of person

**Review by kevinLamKB — APPROVED** [review]

**Review by akd13 — APPROVED** [review]

**Pull request merged**